### DISCUSSION TITLE
2005/05/04/%E5%A6%82%E4%BD%95%E6%B7%BB%E5%8A%A0%E8%AE%A1%E6%95%B0%E5%99%A8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#19] [Announcements]
# 2005/05/04/%E5%A6%82%E4%BD%95%E6%B7%BB%E5%8A%A0%E8%AE%A1%E6%95%B0%E5%99%A8

https://blog.qizhen.xyz/2005/05/04/%E5%A6%82%E4%BD%95%E6%B7%BB%E5%8A%A0%E8%AE%A1%E6%95%B0%E5%99%A8

**Comment by ruanqizhen** [comment]
拉里
2005年08月5日 10:46 编辑
谢谢 前面的我都弄好了 也出现我选的计数器在IE上，可是我就是怎么也不能把它拖到日志上！那个各自占一半的是怎么做？怎么拖拽

回复
Unknown
2005年06月29日 23:35 编辑
看得我眼睛都直拉，好难哦，加油！！

回复
Qizhen
2005年05月7日 12:44 编辑
我这里可以啊，难道被政府禁了？不过没有图也没关系，看下面的说明也可以。

回复
๑•ิ.•ั๑ 兔兔 εїз*
2005年05月7日 12:08 编辑
图1的网址打不开耶。。。

回复
Qizhen
2005年05月5日 12:36 编辑
是的，要把计数器的图片copy到日志里。

回复
Unknown
2005年05月5日 12:28 编辑
照做了,可还是没出现计数器,是写在网志里,然后发布吧?

回复
._小茶ˇ┆
2005年05月5日 00:00 编辑
偶来了.好哦..交换交换!赞!

回复
丹
2005年05月4日 17:15 编辑
以前也常说，戴眼镜好，又挡风挡砂，还能让人觉得我很书生气。嘻嘻，当然是因为我不老实才希望别人觉得我有书生气。现在视力下降，该去配眼镜了，我到反而不想戴眼镜了。人真是个自相矛盾的生物呢。还有谢谢到我的海域，无论是小心还是不小心掉下去的。

回复
Gladys
2005年05月4日 16:23 编辑
真厲害~

回复
國良先生
2005年05月4日 13:33 编辑
親愛的 ruanqizhen　　用心做出屬於自己風格的 blog 就很棒了！加油！ ( ^o^)ρ

回复
